src/utils/_text.scss

```diff
@@ -3,6 +3,13 @@
   background-color: transparent;
   border: 0;
   color: transparent;
-  font: 0/0 a;
+  font: 0 / 0 a;
   text-shadow: none;
 }
+//truncate
+@mixin truncate ($max-width, $text-overflow) {
```

Comment:
Have you considered setting default values for parameters?

Reply:
done